sbom/examples/container_image/build/remove_release_data.py

```diff
@@ -10,11 +10,26 @@
     sbom = json.load(fp)
 
 
-for pkg in sbom["packages"]:
+relationships = sbom["relationships"]
+
+# Find the described packages
+described = [
+    rel["relatedSpdxElement"]
+    for rel in relationships
+    if rel["spdxElementId"] == "SPDXRef-DOCUMENT" and rel["relationshipType"] == "DESCRIBES"
+]
+# Find any packages that are VARIANT_OF the described packages
+variants = [
+    rel["spdxElementId"]
+    for rel in relationships
+    if rel["relatedSpdxElement"] in described and rel["relationshipType"] == "VARIANT_OF"
+]
+print(f"Described: {described}\nVariants: {variants}")
+built = described + variants
+for pkg in [pkg for pkg in sbom["packages"] if pkg["SPDXID"] in built]:
```

Comment:
I'm not sure I understand this change. Is this just trying to filter down the packages to those that might represent images? Would there be other packages not using either DESCRIBES or VARIANT_OF that would be of type `oci`?